quantum/matrix.c

```diff
@@ -23,14 +23,43 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "quantum.h"
 
 #ifdef DIRECT_PINS
-static pin_t direct_pins[MATRIX_ROWS][MATRIX_COLS] = DIRECT_PINS;
-#elif (DIODE_DIRECTION == ROW2COL) || (DIODE_DIRECTION == COL2ROW)
+static const pin_t direct_pins[MATRIX_ROWS][MATRIX_COLS] = DIRECT_PINS;
+#else
+/* Let's default to EITHERWAY. This works regardless of COL2ROW or ROW2COL wiring. */
```

Comment:
I would prefer if there was no default direction. See https://github.com/qmk/qmk_firmware/pull/7915

Reply:
Seeing #7915 I realized that with current master (https://github.com/qmk/qmk_firmware/commit/4eed0331bf0876db0d557f86c7d4ef599ff0e6c4) as of today, this is not being caught.

`#define DIODE_DIRECTION CAL2RAW`

Shall we change COL2ROW to 1 and ROW2COL to 2 and so on?